packages/coinstac-ui/app/render/components/activity-indicator/activity-indicator.css

```diff
@@ -0,0 +1,83 @@
+
+.activity-indicator {
```

Comment:
Copied these rules directly out of _app.scss_. Should maybe clean them up, too. They work.